node-cli/tasks.js

```diff
@@ -55,7 +57,7 @@ exports.createTask = async () => {
     realm.write(() => {
       result = realm.create("Task", {
         _id: new bson.ObjectID(),
-        _partition: "My Project",
+        _partition: partition,
         name: task.name,
         status: task.status,
```

Comment:
Just noticed that we have a bug here. The Inquirer prompt uses the status `"In Progress"` but the data model expects `"InProgress"` (no space). Most of the tutorials use an object/enum to convert between these, but we could also just use this quick and dirty fix in-place.
```suggestion
        status: task.status.replace(/\s/g, '');, // Removes space from "In Progress"
```